### ISSUE TITLE
508 Issue - JAWS cannot read the next line using a "down arrow" key

### ISSUE THREAD

**Issue by wonko-gsa** [GSA/sam-design-system#1415]
**Components Involved**
```
`UsaStepIndicatorModel` from '@gsa-sam/ngx-uswds'
```

**Expected/Actual Behavior**
While using JAWS and navigating the page only with keyboards, JAWS should be able to read texts with a down arrow key even after I used `shift+tab` to go back up to the stepper. 

Here are steps: 
1. While using JAWS, I navigate the Exclusions > SED page with a `tab` key. 
2. When the focus is at the stepper: `3. Review Investigation Results`. I hit the `down arrow` key in the keyboard. Then JAWS will read `Review Investigation Results`.
3. I hit the `down arrow` key again. Then it will read a **heading** while the focus is on `3. Review Investigation Results`.
4. I hit the `down arrow` key again. Then it will read a **description of the heading** while the focus is on `3. Review Investigation Results`.
5. I hit the `down arrow` key again. Then it will read a **label: 'Legal Business Name'** while the focus is on `3. Review Investigation Results`.
6. I hit the `down arrow` key again. Then it will read a **description of the label** while the focus is on `3. Review Investigation Results`.
![image](https://github.com/GSA/sam-design-system/assets/117218338/73694196-be45-4cfb-9b19-08d4e1aca043)
7.  I hit the `down arrow` key again. Then it will say "Legal business name edit required" and the focus will be on the input field.
![image](https://github.com/GSA/sam-design-system/assets/117218338/09e7e183-07e1-41a2-8d24-0272c98085d6)
8. While the focus is on the input field of ` 'Legal Business Name'`, I hit `shift+tab` key. The the focus will go back to the stepper: `3. Review Investigation Results`.
![image](https://github.com/GSA/sam-design-system/assets/117218338/19af6fb8-a82a-4e46-9270-5af8ad24a1de)
9. [**Expected Behavior**] Then I hit the `down arrow` key which I **expect** JAWS to read texts and focus on the input field of legal business as steps `#2` ~ `#7` above. 
10. [**Actual Behavior**] But the page will be scrolled down instead of reading. 

**Your Environment**
Angular Version: 16.2.12
@gsa-sam packages version: 
```
    "@gsa-sam/components": "16.0.13",
    "@gsa-sam/layout": "16.0.8",
    "@gsa-sam/layouts": "16.0.0",
    "@gsa-sam/ngx-uswds": "16.0.1",
    "@gsa-sam/ngx-uswds-icons": "16.0.0",
    "@gsa-sam/sam-formly": "16.0.13",
    "@gsa-sam/sam-material-extensions": "16.0.13",
    "@gsa-sam/sam-styles": "3.0.18",
    "@gsa-sam/shared": "16.0.8",
```

**Ticket Number**
IAEMOD-24736

**Additional context**
Attached a video for demo of the issue:
https://github.com/GSA/sam-design-system/assets/117218338/08ea00e9-cd74-4fdb-b2f1-33210661e80f
